test/e2e/system_connection_test.go

```diff
@@ -266,6 +266,12 @@ var _ = Describe("podman system connection", func() {
 			Expect(session.Out.Contents()).Should(BeEmpty())
 			Expect(session.Err.Contents()).Should(BeEmpty())
 
+			podSession := podmanTest.Podman([]string{"--connection", "QA", "ps"})
+			podSession.WaitWithDefaultTimeout()
+			Expect(podSession).Should(Exit(125))
+			// this means we got to ssh'ing and that the connection was found
+			Expect(podSession.OutputToString()).Should(ContainSubstring("Error: failed to connect: ssh"))
```

Comment:
needs to use ErrorToString to capture stderr.

However if you look at the log this test will not work since you trigger the remote mode which does not allow the --root, --runroot flags, etc... that are set by the test helper. You need to use the same exec.Command logic as above

Reply:
thanks I was wondering what that error was all about